apps/mark/frontend/src/app_root.tsx

```diff
@@ -331,6 +336,27 @@ export function AppRoot({
         )
       : [];
 
+  const updateElectionDefinition = useCallback(
+    (electionDefinition: ElectionDefinition) => {
+      dispatchAppState({
+        type: 'updateElectionDefinition',
+        electionDefinition,
+      });
+    },
+    []
+  );
```

Comment:
I went back and forth on the best approach here: should we remove electionDefinition from the frontend store completely? It's cleaner to do so in a single PR but ultimately to keep scope and diff size down I went with this approach of updating the frontend store every time the backend store changes. This way, places that access the frontend store for `electionDefinition` (which I'm guessing is most screens) are not impacted in this PR.

Reply:
Can you mark whatever local functions etc that will be removed as `@deprecated`?

Reply:
The most important thing to mark deprecated is reading `electionDefinition` from the frontend store. But as far as I can tell there's no function for reading from the frontend store; it's just dereferenced from `appState` and passed from `app_root` to its children via props.

However, based on your feedback I did mark `updateElectionDefinition` as deprecated in https://github.com/votingworks/vxsuite/pull/3278/commits/88bc90d1fcf867bf7e19bf4577317b43c27aa1f0#diff-1cc7f1707c76fa44e5e42e6fd40172387d8b03bfc55bd5fa63b7c2f334c78e6eR339